css/types/basic-shape.json

```diff
@@ -292,19 +292,6 @@
                     }
                   ],
                   "notes": "With the preference enabled, the <code>d</code> SVG presentation attribute, <code>clip-path</code> CSS property, and <code>offset-path</code> CSS property support <code>path()</code>."
```

Comment:
I think it's rather hard to interpret this data, and suspect that simplifying it can lead to Firefox desktop/android data being more similar. Should there be an entry with full support from Firefox 97, or what is still not supported in current Firefox? I think that there can also be an initial entry where the note is the same as for Chrome: "Only supported on the &lt;code>offset-path&lt;/code> property."

Breaking these changes out into its own PR would be good though, it's not going to be a simple removal of a statement.

Reply:
I opened #15942 to attempt to resolve this!